apiserver/deploy/base/kustomization.yaml

```diff
@@ -1,11 +1,21 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-
 namespace: ray-system
-
 resources:
 - apiserver.yaml
-
+patchesJson6902:
+- patch: |-
+    - op: replace
+      path: /spec/template/spec/containers/0/imagePullPolicy
+      value: IfNotPresent
+  target:
+    kind: Deployment
+    name: kuberay-apiserver
+    version: v1
 commonLabels:
-  app.kubernetes.io/name: kuberay
   app.kubernetes.io/component: kuberay-apiserver
+  app.kubernetes.io/name: kuberay
+images:
+- name: kuberay/apiserver
+  newName: kuberay/apiserver
+  newTag: latest
```

Comment:
Is this YAML file intended only for development? If users are expected to deploy the KubeRay API Server using this file, we might need to use nightly as the tag, because `kuberay/apiserver:latest` isn't available on public image registries.

Reply:
I think it just for development, but that being said, let me try to see if I can accomodate both.

Reply:
I have added a local `kustomize.yaml` in [0c5eb2b](https://github.com/ray-project/kuberay/pull/1301/commits/0c5eb2b5ad7461a56ee83a78feb1881ff409b905). 

Let me know if you think I should add `install-nightly` and `uninstall-nightly` targets.